test/test_jit.py

```diff
@@ -6851,6 +6851,17 @@ def foo(a, b):
                     with self.assertRaisesRegex(RuntimeError, 'division by 0'):
                         foo(i, j)
 
+    # Testing bitwise shorthand aug assignment
+    def test_bool_augassign(self):
+        @torch.jit.script
+        def func(a : bool, b : bool):
+            a |= b
+            return a
+        self.assertEqual(func(True, False), True | False)
```

Comment:
```suggestion
        self.checkScript(func, (True, False))
```
Use `checkScript` here and for the three additional tests below. This helper function scripts `func` using both the C++ and Python frontends and checks that the output matches regular Python.

Reply:
I tried using checkScript, but then it complains saying it received a script function instead of class or module.
If I remove the torch declarator and run the test, then it crashes. I'm not sure what I am missing here. Should checkScript not be used when torch declarator is used?

Reply:
Yeah, you need to pass a regular Python function to `checkScript`, not a scripted one. If `checkScript` crashes, that's something worth looking into.

Reply:
Ok, let me investigate this then.